book/src/SUMMARY.md

```diff
@@ -1,21 +1,36 @@
 # Kubernetes/CSI Book
 
-- [Introduction](Home.md)
-- [Using a CSI Driver](UsingCSIDriver.md)
+- [Introduction](introduction.md)
+- [Developing CSI Driver for Kubernetes](developing.md)
+    - [Sidecar Containers](sidecar-containers.md)
+        - [external-provisioner](external-provisioner.md)
+        - [external-attacher](external-attacher.md)
+        - [node-driver-registrar](node-driver-registrar.md)
+        - [cluster-driver-registrar](cluster-driver-registrar.md)
+        - [external-snapshotter](external-snapshotter.md)
+        - [livenessprobe](livenessprobe.md)
+    - [CSI CRDs](csi-crds.md)
+      - [CSIDriver Object](csi-driver-object.md)
+      - [CSINodeInfo Object](csi-noide-info-object.md)
+    - [Features](features.md)
+        - [Secrets & Credentials](ephemeral-local-volumes.md)
+        - [Volume Snapshot & Restore](Implement-Snapshot-Feature.md)
+        - [Topology](topology.md)
+        - [Raw Block Volume](block.md)
+        - [Skip Attach](skip-attach.md)
+        - [Pod Info on Mount](pod-info.md)
+        - [Ephemeral Local Volumes](ephemeral-local-volumes.md)
+- [Deploying CSI Driver on Kubernetes](deploying.md)
```

Comment:
"Deploying a"?

Reply:
Wanted to minimize the length of titles in the nav pane. But eh ok 2 more chars won't be too bad.